marklogic-data-hub/src/main/resources/ml-modules/root/data-hub/5/impl/step.sjs

```diff
@@ -77,20 +77,20 @@ class Step {
   }
 
   getStepsByType(type = 'custom') {
-    let query = [cts.collectionQuery('http://marklogic.com/data-hub/step-definition'), cts.jsonPropertyValueQuery('type', type, 'case-insensitive')];
+    let query = [cts.collectionQuery('http://marklogic.com/data-hub/step-definition'), cts.jsonPropertyValueQuery('type', type, ["unstemmed","case-insensitive"])];
     return cts.search(cts.andQuery(query)).toArray();
   }
 
   getStepByNameAndType(name, type = 'custom') {
-    let query = [cts.collectionQuery('http://marklogic.com/data-hub/step-definition'), cts.jsonPropertyValueQuery('name', name, 'case-insensitive'), cts.jsonPropertyValueQuery('type', type, 'case-insensitive')];
+    let query = [cts.collectionQuery('http://marklogic.com/data-hub/step-definition'), cts.jsonPropertyValueQuery('name', name, ["unstemmed","case-insensitive"]), cts.jsonPropertyValueQuery('type', type, ["unstemmed","case-insensitive"])];
```

Comment:
To support a query like this that goes beyond just artifact name, I think we should have another helper function in artifacts/core:

    artifactPropertyValueQuery(name, value)

And that can default to unstemmed / case-insensitive. 

The goal is to limit the number of places where we're coding "unstemmed" as a query option. This query would then become:

    hubUtils.findDocuments(cts.andQuery([artifactPropertyValueQuery("name", name), artifactPropertyValueQuery("type", type)]));